quackstack/ducky.py

```diff
@@ -31,7 +31,8 @@ class DuckBuilder:
         filename.stem: filename for filename in (ASSETS_PATH / "accessories/outfits").iterdir()
     }
 
-    def __init__(self):
+    def __init__(self, seed: int = None):
+        self.random = Random(seed) if seed else Random()
```

Comment:
Can't you just leave this as `Random(seed)`? The docs say the seed can be `NoneType`.